app.js

```diff
@@ -75,7 +78,7 @@ app.get('/components/:component', function (req, res, next) {
   // make variables available to nunjucks template
   res.locals.componentPath = req.params.component
 
-  res.render(`${req.params.component}/index`, function (error, html) {
+  res.render('../../' + configPaths.components + `${req.params.component}/index`, function (error, html) {
```

Comment:
Why do we need to specify the entire path if `configPaths.components` is in the appViews?